lib/browser/guest-view-manager.js

```diff
@@ -328,8 +328,8 @@ const isWebViewTagEnabledCache = new WeakMap()
 
 const isWebViewTagEnabled = function (contents) {
   if (!isWebViewTagEnabledCache.has(contents)) {
-    const value = contents.getLastWebPreferences().webviewTag
-    isWebViewTagEnabledCache.set(contents, value)
+    const webPreferences = contents.getLastWebPreferences() || {}
+    isWebViewTagEnabledCache.set(contents, webPreferences.webviewTag)
```

Comment:
Why does it make sense to return `undefined` in some cases?
The function is supposed to return a boolean value.

Reply:
fixed